ComfortZoneApp/app/src/main/java/com/example/comfortzone/fragments/FlightFragment.java

```diff
@@ -107,6 +109,28 @@ private void setObjects() {
         spinnerAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
     }
 
+    private void checkCities() {
+        Observable<Object> ob = WeatherDbUtil.maybeUpdateCitiesList(getActivity());
+        Subscriber mySubscriber = new Subscriber() {
```

Comment:
I think we can be more descriptive about the names for Observable and Subscriber so it's easier to identify

i.e. `dataSetupObservable` & `dataSetupSubscriber`